tests/Auth/FileUserTest.php

```diff
@@ -120,4 +120,28 @@ public function it_gets_permissions_from_a_cache()
         // Doing it a second time should give the same result but without multiple calls.
         $this->assertEquals($expectedPermissions, $user->permissions()->all());
     }
+
+    /**
+     * @test
+     */
+    public function it_has_a_dirty_state()
```

Comment:
For full coverage, can we add a similar test to `EloquentUserTest`?

Reply:
Yep, done